### PR TITLE
feat(global): added light and dark themes

### PR DESCRIPTION
**Pull request by rakshit087**
We can extend this component with colors, also a media query can be used to auto update the theme.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,30 @@
-import { useState } from 'react'
+import { CssBaseline, createTheme, ThemeProvider } from '@mui/material'
 
 function App() {
```
Comment (pulkitvyas08) [inline src/App.tsx:3]: Convert this to an arrow function

File: src/App.tsx [hunk 2]
```diff
@@ -1,10 +1,30 @@
...
+      background: {
+        default: '#FFFFFF',
+      }
+    },
+  })
   return (
-    <div className="App">
-      <p>Hello World</p>
-    </div>
+    <ThemeProvider theme={darkTheme}>
```
Comment (pulkitvyas08) [inline src/App.tsx:22]: Are we really going for this? Will this make #3 difficult to implement?

**Review by pulkitvyas08 — CHANGES_REQUESTED** [review]

**Comment by pulkitvyas08** [comment]
@rakshit087 Are you doing Varcel or should I?

File: src/App.tsx (hunk 2 above)
Comment (rakshit087) [reply src/App.tsx:22]: I thought it’s the same thing. Material has some great defaults for dark theme and all we need to do is toggle a state in order to switch mode.
https://mui.com/customization/dark-mode/

**Comment by rakshit087** [comment]
> @rakshit087 Are you doing Varcel or should I?

Yes please do :))

File: src/App.tsx (hunk 2 above)
Comment (pulkitvyas08) [reply src/App.tsx:22]: Ohkay makes sense so we can use the state that they would provide. Great then go ahead!

**Comment by pulkitvyas08** [comment]
> > @rakshit087 Are you doing Varcel or should I?
> 
> Yes please do :))

Will do!

**Review by pulkitvyas08 — APPROVED** [review]

**Pull request merged**